.github/workflows/apt-release.yml

```diff
@@ -9,13 +9,11 @@ jobs:
     steps:
     - name: Check Out Repo
       uses: actions/checkout@v2
-      
+
     - name: Configure AWS credentials
       uses: aws-actions/configure-aws-credentials@v1
       with:
-        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-        aws-session-token: ${{ secrets.AWS_SESSION_TOKEN }}
+        role-to-assume: ${{ secrets.AWS_ACCESS_ROLE}}
```

Comment:
What is the maintenance for this role? Would we have to rotate/re-generate it periodically?